cli/smartcontract/smart_contract.go

```diff
@@ -172,6 +172,10 @@ func NewCommands() []cli.Command {
 						Name:  "no-permissions",
 						Usage: "do not check if invoked contracts are allowed in manifest",
 					},
+					cli.BoolFlag{
+						Name:  "no-nef-check",
```

Comment:
@roman-khimov, what do you think about this option? You told earlier that it's hardly possible that someone needs large compiled contract that can not be deployed, so remove it?